__main__.py

```diff
@@ -60,6 +65,12 @@ def load_map(self):
             tile_map, "floor", TILE_SPRITE_SCALING, use_spatial_hash=True
         )
 
+        # Object Layers
+
+        self.object_layers = utils.process_objects(f"assets/tilemaps/TestLevel.tmx")
+
+        self.guard1_locations = utils.extract_guard_locations(self.object_layers[0])
```

Comment:
Is there a way to iterate through this and add each guard's location into a list. Rather than doing each one, one by one.